packages/ckeditor5-ui/tests/dropdown/dropdownview.js

```diff
@@ -469,4 +470,42 @@ describe( 'DropdownView', () => {
 			} );
 		} );
 	} );
+
+	describe( 'on dropdown close', () => {
+		it( 'should focus a dropdown button if focus is inside the dropdown', () => {
+			// Create a button inside the dropdown panel to enable focus.
+			view.panelView.children.add( new ButtonView( locale ) );
+
+			view.isOpen = true;
+
+			expect( view.panelView.element.contains( document.activeElement ) ).to.be.true;
+
+			view.isOpen = false;
+
+			expect( view.element.contains( document.activeElement ) ).to.be.true;
+		} );
+
+		it( 'should not focus dropdown button if focus is outside the dropdown', () => {
+			// Setup an element that is not a child of the dropdown to be focused.
+			const button = document.body.appendChild( document.createElement( 'button' ) );
+
+			button.classList.add( 'test' );
+
+			// Create a button inside the dropdown panel.
+			view.panelView.children.add( new ButtonView( locale ) );
+
+			view.isOpen = true;
+
+			expect( view.panelView.element.contains( document.activeElement ) ).to.be.true;
+
+			document.querySelector( '.test' ).focus();
```

Comment:
What if there's some test runner element with the same class? You could've saved the reference to the DOM element in L490 and re-used it here. Also, no reason to give a specific class then.